bundles/binding/org.openhab.binding.panstamp/src/main/java/org/openhab/binding/panstamp/internal/PanStampBinding.java

```diff
@@ -0,0 +1,566 @@
+/**
+ * Copyright (c) 2010-2015, openHAB.org and others.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.panstamp.internal;
+
+import java.io.File;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
+
+import me.legrange.panstamp.Endpoint;
+import me.legrange.panstamp.EndpointListener;
+import me.legrange.panstamp.EndpointNotFoundException;
+import me.legrange.panstamp.ModemException;
+import me.legrange.panstamp.Network;
+import me.legrange.panstamp.NetworkException;
+import me.legrange.panstamp.NetworkListener;
+import me.legrange.panstamp.PanStamp;
+import me.legrange.panstamp.PanStampListener;
+import me.legrange.panstamp.Register;
+import me.legrange.panstamp.RegisterListener;
+import me.legrange.panstamp.definition.CompoundDeviceLibrary;
+import me.legrange.panstamp.event.AbstractNetworkListener;
+import me.legrange.panstamp.event.AbstractPanStampListener;
+import me.legrange.panstamp.event.AbstractRegisterListener;
+import me.legrange.panstamp.xml.ClassLoaderLibrary;
+import me.legrange.panstamp.xml.FileLibrary;
+import me.legrange.swap.ModemSetup;
+import me.legrange.swap.SerialModem;
+import me.legrange.swap.SwapException;
+import me.legrange.swap.SwapModem;
+import me.legrange.swap.tcp.TcpException;
+import me.legrange.swap.tcp.TcpServer;
+
+import org.openhab.binding.panstamp.PanStampBindingConfig;
+import org.openhab.binding.panstamp.PanStampBindingProvider;
+import org.openhab.core.binding.AbstractActiveBinding;
+import org.openhab.core.binding.AbstractBinding;
+import org.openhab.core.binding.BindingChangeListener;
+import org.openhab.core.binding.BindingProvider;
+import org.openhab.core.types.Command;
+import org.openhab.core.types.State;
+import org.osgi.framework.BundleContext;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * This binding connects to a panStamp network and handles traffic to and from
+ * it.
+ * 
+ * @author Gideon le Grange
+ * @since 1.8.0
+ */
+public class PanStampBinding extends
+		AbstractBinding<PanStampBindingProvider> {
+
+	private static final Logger logger = LoggerFactory
+			.getLogger(PanStampBinding.class);
+	private PanStampBindingSettings cfg;
+	private SwapModem modem;
+	private Network network;
+	private ConfigDeviceStore store;
+	private TcpServer debugServer;
+
+	/**
+	 * @{inheritDoc
+	 */
+	public void activate(final BundleContext bundleContext,
+			final Map<String, Object> configuration) {
+		logger.debug("activate()");
+		try {
+
+			startup(PanStampBindingSettings.parseConfig(configuration));
+		} catch (ValueException e) {
+			logger.error(e.getMessage());
```

Comment:
The error messages generated by ```parseConfig()``` should usually be descriptive and tell the user which setting failed for what reason. I am changing this to: 

```java
catch (ValueException e) {
			logger.error("Configuration error: {} ", e.getMessage());
}
```
which should lead to error messages like ```Configuration error: Invalid network.channel (-1000 is out of range)```